### ISSUE TITLE
make logo more accessible

### ISSUE THREAD

**Issue by artfulrobot** [mastodon/share#12]
This:

![screenshot of mastodon logo](https://github.com/user-attachments/assets/88626355-3ae2-4f0b-9483-153a7b505c26)

has black lettering paced over a dark purple icon making it hard to read by people who need minimum contrast ratios. It is certainly not  even at WCAG AA standard.

It could be fixed by removing the overlap or lightening the icon.

Thanks for the software though!

**Comment by Gargron** [comment]
The logo shape or colors [cannot be changed](https://joinmastodon.org/branding).
